### PR TITLE
Development preview branch

### PR DESCRIPTION
**Pull request by segfaultxavi**

### CODE REVIEW

File: source/guides/network/supernode-enrollment.rst [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+   This transaction pays a small :doc:`fee <../../concepts/fees>` so make sure your main account has **extra funds** beyond the requirements of the Supernode program.
+
+******************
+Validate the setup
+******************
+
+You can **validate your node** by checking that all services are running properly. Check that the following URLs return valid data:
+
+* `localhost:3000/chain/info <https://localhost:3000/chain/info>`__: Node's connection to the network.
```
Comment (fboucquez) [inline source/guides/network/supernode-enrollment.rst:165]: HTTP (wihtout s) when port is 3000

File: source/guides/network/supernode-enrollment.rst [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+First off, make sure your node is up and running (or ready to run). `This guide <https://github.com/nemtech/catapult-server/blob/main/docs/RUNPEERLIN.md>`__ explains how to setup the ``catapult-server`` service, but you will need other services as well to fully configure a node (**Guide coming soon**).
+
+The process requires **installing a monitoring agent** and **announcing a special transaction**. You are going to create a few files, so please create a folder to keep your filesystem tidy.
+
+.. note:: The following instructions are for |codename|'s test network (TESTNET). Once |codename| launches they will be updated for the main network (MAINNET).
+
+1. **Download the node monitoring agent**:
+
+   `Download the Linux binary <https://symbol-node-reward.s3-eu-west-1.amazonaws.com/agent_binary/agent-linux.bin>`__ and make it executable:
```
Comment (fboucquez) [inline source/guides/network/supernode-enrollment.rst:81]: The new url will be https://symbol-node-reward.s3-eu-west-1.amazonaws.com/packages/0.1.0/agent_binary/agent-linux.bin

Can you add the version to a variable?

Comment (segfaultxavi) [reply source/guides/network/supernode-enrollment.rst]: You mean a variable for us, that we can update, or a placeholder for the user so he has to find out what the latest version is and build the right URL? The latest sounds uncomfortable.

File: source/guides/network/supernode-enrollment.rst (hunk 1 above)
Comment (segfaultxavi) [reply source/guides/network/supernode-enrollment.rst]: 👍

**Pull request merged**